src/store/modules/notification/index.ts

```diff
@@ -0,0 +1,152 @@
+import { NOTIFICATION_UNKNOWN_ERROR } from '@/setup/globals';
+import i18n from '@/setup/i18n';
+import api from '@/helpers/api';
+import { IMessage, INotification } from '@/types/c-notification';
+import { defineModule } from 'direct-vuex';
```

Comment:
Please always import node modules before local files.

Reply:
This has been reworked in the develop and already replaced in the remake branch